### PR TITLE
L10n pt updates 2023 11 17

### PR DESCRIPTION
**Pull request by steppi**
#### Brief description of what is fixed or changed
This PR adds recent updates made to the Portuguese translations. Similar to gh-698, to handle lack of synchrony between Crowdin and the actually published translations, I did a scripted rebase which filtered to just portuguese commits, and squashed all commits together which had touched the same file. I then rebased on main and manually fixed the merge conflicts. @melissawm, please let me know if everything looks OK.

Also, it appears XND has been replaced by Awkward Array in the list of dependent projects in the table of contents. The description "Manipulate JSON-like data with NumPy-like idioms." is currently untranslated.

### CODE REVIEW

File: content/pt/news.md [hunk 1]
```diff
@@ -1,17 +1,32 @@
...
 
-### numpy.org agora está disponível em Japonês e Português
+### Lançado o NumPy versão 1.26.0
 
-_02 de agosto, 2023_ -- Estamos felizes em anunciar que o site numpy.org agora está disponível em duas línguas adicionais além do Inglês: Japonês e Português Brasileiro. Isso não teria sido possível sem nosso grupo de voluntários: 
+_16 de setembro de 2023_ -- [NumPy 1.26.0](https://numpy.org/doc/stable/release/1.26.0-notes.html) está disponível. Os destaques desta versão são:
 
-Português:
-* Melissa Weber Mendonça (melissawm)
+* Suport ao Python 3.12.0.
```
Comment (melissawm) [inline content/pt/news.md:12]:
```suggestion
* Suporte ao Python 3.12.0.
```

File: content/pt/tabcontents.yaml [hunk 2]
```diff
@@ -75,14 +75,15 @@ params:
         alttext: xtensor
         url: https://github.com/xtensor-stack/xtensor-python
       - 
-        title: XND
-        text: Desenvolva bibliotecas para computação com arrays, recriando conceitos fundamentais do NumPy.
-        img: /images/content_images/arlib/xnd.png
-        alttext: xnd
-        url: https://xnd.io
+        title: Awkward Array
+        text: Manipulate JSON-like data with NumPy-like idioms.
```
Comment (melissawm) [inline content/pt/tabcontents.yaml:79]: Manipulação de dados JSON-like com sintaxe NumPy-like.

**Review by melissawm — COMMENTED** [review]
Not sure if these changes should be made directly here or on crowdin for consistency, let me know if that's the case and I'll do them there as wel. Thanks!

**Comment by steppi** [comment]
Thanks @melissawm. I think it's fine to make the changes here first. I already had to do some git surgery and fix merge conflicts and I'd have to do some of that again if we go through Crowdin. Later you can go back and make the changes in Crowdin too to keep merge conflicts from happening on the next PR. I think the Crowdin CLI allows syncing translations back from the repo to Crowdin, but I haven't figured out how to do it without causing other problems yet; still looking into it.

I'll try to figure out why the build failed.

File: content/pt/news.md [hunk 3]
```diff
@@ -1,17 +1,32 @@
...
+
+As versões do Python suportadas por esta versão são 3.9-3.12.
+
+### numpy.org agora está disponível em japonês e português
+
+_2 de agosto de 2023_ -- numpy.org agora está disponível em 2 idiomas adicionais: japonês e português. Isto não seria possível sem nossos voluntários dedicados:
+
+_Português:_
+* Melissa Weber Mendonça (melissawm)
+>>>>>>> 2ea4033 (New translations news.md (Portuguese, Brazilian))
```
Comment (stefanv) [inline content/pt/news.md:29]: Merge artifact?

**Comment by steppi** [comment]
Thanks @melissawm, I think this is ready now. Thanks @stefanv for pointing out the merge artifact.

**Review by stefanv — APPROVED** [review]
Looks good, thanks!

**Pull request merged**